src/covid19model/models/models.py

```diff
@@ -149,36 +152,45 @@ def integrate(t, S, E, I, A, M, ER, C, C_icurec, ICU, R, D, H_in, H_out, H_tot,
         # calculate total population
         # ~~~~~~~~~~~~~~~~~~~~~~~~~~
         T = S + E + I + A + M + ER + C + C_icurec + ICU + R + V
-        vacc_eligible = S + R + E + I + A
+        # vaccination eligible states
+        VE = S + R + E + I + A
```

Comment:
Is it necessary to compute VE here? In terms of bookkeeping, VE is a state which is updated using the differential dVE, so this doesn't seem necessary.

Reply:
true